src/test/java/core/basesyntax/strategy/operation/ReturnHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.strategy.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.FruitTransaction.Operation;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ReturnHandlerTest {
+    private static String FRUIT = "apple";
+    private OperationHandler returnHandler;
+
+    @Before
+    public void setUp() {
+        returnHandler = new ReturnHandler();
+    }
+
+    @Test
+    public void addReturnHandler() {
+        Storage.fruits.put(FRUIT, 0);
+        FruitTransaction returnTransaction = new FruitTransaction(
+                Operation.getByCode("r"), FRUIT, 20);
```

Comment:
```suggestion
                Operation.RETURN, FRUIT, 20);
```
just use enum instead of method

Reply:
Thank you @aaananas, I fixed this in all Handler Tests. Fixes were made according to the comments above too.